tests/cursorid/cursorid-set_state_error-001.phpt

```diff
@@ -7,12 +7,12 @@ require_once __DIR__ . '/../utils/basic.inc';
 
 echo throws(function() {
     MongoDB\Driver\CursorId::__set_state(['id' => 0]);
-}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
+}, 'MongoDB\Driver\Exception\InvalidArgumentException::class'), "\n";
```

Comment:
Apologies if I didn't clarify in https://github.com/mongodb/mongo-php-driver/pull/1231#discussion_r658990539, but `::class` is actually a built-in class constant that resolves to a string value. Here, you're actually expecting an exception with the class name "MongoDB\Driver\Exception\InvalidArgumentException::class", which doesn't exist. Please remove the quotes.

The "ALMOST" below is telling us that an exception _was_ thrown but it didn't match the class we expected. That output is just to make the test diffs more readable and isn't something we should ever expect. For assertions such as these, we'd only ever want to see "OK" in the test output for an exact match.

Reply:
Oh sorry got it, just fixed!